sway/ipc-server.c

```diff
@@ -644,6 +644,8 @@ void ipc_client_handle_command(struct ipc_client *client) {
 			json_object_object_add(output_json, "focused",
 				json_object_new_boolean(focused));
 
+			const char *subpixel = sway_wl_output_subpixel_to_string(output->wlr_output->subpixel);
+			json_object_object_add(output_json, "subpixel hinting", json_object_new_string(subpixel));
```

Comment:
Can you  please also change this to `subpixel` or `subpixel_hinting` and document it in `sway-ipc.7.scd`? (swaymsg pretty print will also need to be updated to reflect the change)

Reply:
Done.